airflow/dags/geo_reference/load_building_footprints.py

```diff
@@ -22,11 +22,11 @@ def building_footprints_dag():
     """DAG for loading MS Building footprints dataset."""
     submit_batch_job = BatchOperator(
         task_id="load_footprints",
-        job_name="california_building_footprints",
+        job_name="us_building_footprints",
```

Comment:
@britt-allen with this PR there are two different building footprints datasets. I've tried to go through and use consistent naming for them, following the actual Microsoft repos. The older one is called `us_building_footprints`, and the newer one is called `global_ml_building_footprints`

Reply:
thank you for updating the names to be more consistent with Microsoft and more descriptive